### PR TITLE
fix(commitlint): #242 fix pinned dependencies

### PR DESCRIPTION
**Pull request by mike-winberry**
## Description
#242 

- [x] create package(-lock).json w/ dev deps needed
- [x] remove the install commitlint step from commitlint action

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://github.com/defenseunicorns/go-oscal/blob/main/CONTRIBUTING.md) followed

### CODE REVIEW

File: package.json
Comment (brandtkeller) [inline package.json:1]: Is it a requirement that these live at the root of the project?

Comment (mike-winberry) [reply package.json:1]: typically yes, thats where the package(-lock) lives, I can look into using a seperate folder and see if that works.

**Comment by brandtkeller** [comment]
I see the following in the action runtime: 

```
npm warn exec The following package was not found and will be installed: commitlint@19.3.0
```

**Comment by mike-winberry** [comment]
> I see the following in the action runtime:
> 
> ```
> npm warn exec The following package was not found and will be installed: commitlint@19.3.0
> ```

Looks like it is an issue with the alias attempting a fix.

**Review by brandtkeller — APPROVED** [review]

**Pull request merged**